lms/djangoapps/teams/views.py

```diff
@@ -1055,7 +1055,8 @@ def get(self, request):
                     CourseAccessRole.objects.filter(user=request.user, role='staff').values_list('course_id', flat=True)
                 )
                 accessible_course_ids = [item for sublist in (enrolled_courses, staff_courses) for item in sublist]
-                if requested_course_id is not None and requested_course_id not in accessible_course_ids:
+                if requested_course_id is not None \
+                        and CourseKey.from_string(requested_course_id) not in accessible_course_ids:
```

Comment:
If `requested_course_id` is not `None` at this point, can't we just use `requested_course_key` instead of re-deriving it?